index.js

```diff
@@ -2,13 +2,30 @@ const express = require('express');
 const helmet = require('helmet');
 const cors = require('cors');
 
+const usersRoutes = require('./users/usersRoutes');
+
 const server = express();
 
 server.use(helmet());
 server.use(cors());
 server.use(express.json());
 server.use(express.urlencoded({ extended: true }));
 
```

Comment:
Why do we need this middleware?

Reply:
With cors middleware being present, setting the header was not really needed